### PR TITLE
feature(hydra-prepare-region): add ability to pre-configure region in…

### PR DESCRIPTION
**Pull request by bentsi**
… AWS

the feature allowes automatically creating all the resources needed
for SCT test run environment. This includes configuraiton of VPC,
subnet, SG and other resources.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
@bentsi 
I've run into `aws-eu-west2-qa-builder1` being misconfigured for running scylla docker, can we tweak this parameter to be able to run docker backend if needed (i.e. the artifact test was failing each time it's hitting this worker)
```bash
jenkins@ip-10-0-2-16:~$ cat /proc/sys/fs/aio-max-nr
65536
```

**Comment by bentsi** [comment]
Successful run of longevity pipeline: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/bentsi-staging/job/bentsi-longevity-3h/5/console

File: test-cases/bentsi.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+test_duration: 10
```
Comment (enaydanov) [inline test-cases/bentsi.yaml:1]: Is this file here accidentally?

Comment (bentsi) [reply test-cases/bentsi.yaml:1]: I will remove it before the merge. Used to test the PR on by job....

**Pull request merged**